src/MLOps.NET.AWS/MLOps.NET.AWS.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
ℹ️ Dev note: This was an active choice. If we're going to use EF Core with migrations we need to either use net472 or netcoreapp3.1. I didn't feel like we should have to multi-target this framework for both, so I felt it was best to make an active choice only to support .NET Core which I think is fine